ross/available_materials.toml

```diff
@@ -1,9 +1,9 @@
 [Materials.Steel]
 name = "Steel"
-rho = 7810
+rho = 7810.0
 E = 211000000000.0
-Poisson = 0.2992610837438423
 G_s = 81200000000.0
+Poisson = 0.2992610837438423
```

Comment:
I think these changes are not related to this PR.
Maybe this file got changed when running the doctests.
When running doctests we are creating several files and also changing the available_materials file. 
I have open an issue for that #590.